### DISCUSSION TITLE
Unraid Madmax error

### DISCUSSION THREAD

**Discussion by JCandle** [guydavis/machinaris#195] [Q&A]
Hi,

I keep getting the below error when using madmax and ramdisk. I have 160GB of ram, so it shouldn’t be an issue but it seems once Madmax uses half my ram the Error populates. Any ideas on how to solve this?

thanks. 
![3EDF8926-411F-41D0-910A-21B8E74CDCF2](https://user-images.githubusercontent.com/32656641/126053787-2f6b3903-a2de-43df-8112-28fa0c98fbe2.png)
![C2F37594-841E-498C-9299-6736FA3D6F0A](https://user-images.githubusercontent.com/32656641/126053791-736a9abd-7cea-4b1d-90f4-7c3a713a4422.png)

**Comment by guydavis** [comment]
Hi, sorry to hear you're encountering this.  Perhaps the issue is like [this one](https://github.com/madMAx43v3r/chia-plotter/issues/561) reported against the plotter itself?

**Comment by JCandle** [comment]
Running an R720 so I can’t manually turn up the voltage…

**Reply by guydavis** [reply]
Hmm, maybe an memory test at the [Unraid boot menu](https://forums.unraid.net/topic/74167-best-way-to-test-new-ram-for-stability/?do=findComment&comment=683123)?

**Reply by JCandle** [reply]
Tried that no issues. Does it have something to do with having dual cpus and half the ram being on each cpu? I don’t know enough about this stuff.

**Reply by guydavis** [reply]
Okay, maybe a disk full like [this](https://github.com/madMAx43v3r/chia-plotter/issues/800) or [this](https://github.com/madMAx43v3r/chia-plotter/issues/548).  FYI, latest `:test` version of Machinaris now also pre-clears out the `tmp2` space as well.  Try [updating](https://github.com/guydavis/machinaris/wiki/Releases).
